pkg/json_raw_results.go

```diff
@@ -137,8 +138,28 @@ type jsonLicense struct {
 	// TODO: add fields, like type of license, etc.
 }
 
+type jsonWorkflows struct {
+	UntrustedCheckouts   []jsonUntrustedCheckout `json:"untrustedCheckouts"`
+	ScriptInjections     []jsonScriptInjection   `json:"scriptInjections"`
+	SecretInPullRequests []jsonEncryptedSecret   `json:"secretInPullRequests"`
+}
+
+type jsonUntrustedCheckout struct {
```

Comment:
shall I simplify here and make each of these `jsonUntrustedCheckout`, et a `jsonFile`?